app/controllers/middleware_server_controller.rb

```diff
@@ -8,13 +8,42 @@ class MiddlewareServerController < ApplicationController
   after_action :set_session_data
 
   OPERATIONS = {
-    :middleware_server_reload => {:op   => :reload_middleware_server,
-                                  :hawk => N_('Not reloading Hawkular server'),
-                                  :msg  => N_('Reload initiated for selected server(s)')
+    :middleware_server_reload  => {:op    => :reload_middleware_server,
+                                   :skip  => true,
+                                   :hawk  => N_('reloading'),
+                                   :msg   => N_('Reload')
     },
-    :middleware_server_stop   => {:op   => :stop_middleware_server,
-                                  :hawk => N_('Not stopping Hawkular server'),
-                                  :msg  => N_('Stop initiated for selected server(s)')
+    :middleware_server_stop    => {:op    => :stop_middleware_server,
+                                   :skip  => true,
+                                   :hawk  => N_('stopping'),
+                                   :msg   => N_('Stop')
+    },
+    :middleware_server_restart => {:op    => :restart_middleware_server,
+                                   :skip  => true,
+                                   :hawk  => N_('restarting'),
+                                   :msg   => N_('Restart')
+    },
+    :middleware_server_shutdown => {:op   => :shutdown_middleware_server,
+                                   :skip  => true,
+                                   :hawk  => N_('shutting down'),
+                                   :msg   => N_('Shutdown'),
+                                   :param => ':timeout'
+    },
+    :middleware_server_suspend => {:op    => :suspend_middleware_server,
+                                   :skip  => true,
+                                   :hawk  => N_('suspending'),
+                                   :msg   => N_('Suspend'),
+                                   :param => ':timeout'
+    },
+    :middleware_server_resume  => {:op    => :resume_middleware_server,
+                                   :skip  => true,
+                                   :hawk  => N_('resuming'),
+                                   :msg   => N_('Resume')
+    },
+    :middleware_server_jdr     => {:op    => :create_jdr_report,
```

Comment:
yes I agree that trigger is the same. but indeed it's async, and also I'm not sure the user would be expecting a report generation in the same menu as power operations. so ux wise it might be better placed somewhere else.